design_introspection-plugin/tests/get_cells/get_cells.tcl

```diff
@@ -1,7 +1,6 @@
 yosys -import
-plugin -i design_introspection
-#Import the commands from the plugins to the tcl interpreter
-yosys -import
+if { [info procs get_cells] == {} } { plugin -i design_introspection }
+yosys -import  ;# ingest plugin commands
```

Comment:
Please put both the plugin load and import behind the if.

Reply:
I considered that initially ,but there are a bunch of files that actually use multiple plugins, so I standardized on loading all the plugins, then finally do a `yosys -import`.

Also, it keeps the plugin load lines nice one-liners which I think makes for easier copy-paste of this boilderplate code later.

```
if { [info procs get_cells] == {} } { plugin -i design_introspection; yosys -import }
```
... would be 86 characters long..

Would there be a good reason to do `yosys -import` anyway within the `if` ?